packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -91,7 +91,8 @@ class CreateCommand extends FlutterCommand {
       abbr: 's',
       help: 'Specifies the Flutter code sample to use as the main.dart for an application. Implies '
         '--template=app. The value should be the sample ID of the desired sample from the API '
-        'documentation website (http://docs.flutter.io).',
+        'documentation website (http://docs.flutter.dev). An example can be found at '
+        'https://master-api.flutter.dev/flutter/widgets/SingleChildScrollView-class.html',
```

Comment:
Shall we use https://api.flutter.dev/flutter/widgets/SingleChildScrollView-class.html instead of https://master-api.flutter.dev/flutter/widgets/SingleChildScrollView-class.html ?

Reply:
That doesn't have the sample code section yet

Reply:
I see. LGTM. Maybe create some issue that reminds us to change back to "api.flutter.dev" once the sample code is there.